README.md

```diff
@@ -385,6 +385,35 @@ You will need:
 - `--config-template-path` option to provide a path to a config file.
 - `--config-type` option specify which default template to use to create the config file.
 
+## Cancel pending transaction
+This command will cancel pending transaction.
+
+Please note that a this action is irreversible.
+
+You will need:
+
+- `--nonce` option specify which transaction to cancel.
+- `--gas-price` option, the gas price is required to be higher than the pending transaction.
+- `--transaction-hash` transaction hash option can be used as an alternative to nonce and gas-price option.
```

Comment:
Let's add a quick explanation on what we do in this case (we automatically add 10% higher gas free)

Reply:
okay, 50% higher gas price.